subsys/bluetooth/host/hci_core.h

```diff
@@ -481,3 +481,11 @@ void bt_hci_synchronous_conn_complete(struct net_buf *buf);
 
 void bt_hci_le_df_connection_iq_report(struct net_buf *buf);
 void bt_hci_le_df_cte_req_failed(struct net_buf *buf);
+
+#if defined(CONFIG_BT_CTLR_LE_POWER_CONTROL)
+void bt_hci_le_transmit_power_report(struct net_buf *buf);
+#endif
+
+#if defined(CONFIG_BT_CTLR_PATH_LOSS_MONITORING)
+void bt_hci_le_path_loss_report(struct net_buf *buf);
+#endif
```

Comment:
Do not conditional compile function prototypes in headers.

```suggestion
void bt_hci_le_transmit_power_report(struct net_buf *buf);

void bt_hci_le_path_loss_report(struct net_buf *buf);
```